app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -437,7 +439,11 @@ class GleanMetricsService(private val context: Context) : MetricsService {
         // can handle events being recorded before it's initialized.
         starter = MainScope().launch {
             Glean.registerPings(Pings)
-            Glean.initialize(context, Configuration(channel = BuildConfig.BUILD_TYPE))
+            Glean.initialize(context,
+                Configuration(channel = BuildConfig.BUILD_TYPE,
+                    httpClient = ConceptFetchHttpUploader(
+                        lazy { GeckoViewFetchClient(context) }
```

Comment:
I think we want to use our existing client instance here `lazy { context.components.core.client }` or is there a reason you created a new instance?

Reply:
Oh, no I didn't do that deliberately. Sure, I can fix that easily.

Reply:
OK cool, let's land once you get a chance to push up the change.

Reply:
I'm just waiting for master to become unbroken and then I'll land it.

Reply:
@csadilek It's been landed now.